main.py

```diff
@@ -150,19 +150,14 @@ def make_app():
             )
             raise
 
-        # Get build date for login notifications
-        with open("build.date") as f:
-            timestamp = dt.utcfromtimestamp(int(f.read()))
-            glob.latestBuild = timestamp.strftime("%b %d %Y")
-
```

Comment:
deleted these bad boys for stateless; need to remember to remove from sql @cmyui